CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 _Version 0.5.2_:
 
  * [serlib] Compatibility with OCaml 4.07.0 [problems with `Stdlib` packing]
+ * [sertop] Support for `-I` option.
```

Comment:
Please also add the long name `--ml-include-path` which should be auto-explanatory.

Reply:
Version `0.5.2` has already been released, please open a new `0.5.3` item.